src/Entities/Question.php

```diff
@@ -8,15 +8,17 @@ class Question extends BaseEntity
     private $type;
     private $data;
     private $widget_type;
```

Comment:
I know the code originally had `$widget_type` in it, so this isn't really part of your change - but since it's private (so not impacting other classes) and you're modifying the same file anyway, can we change this to be camel case for consistency with everything else? The methods can remain snake case.

Reply:
Done